jobs/send-stale-initial-pr-reminder.js

```diff
@@ -11,13 +12,24 @@ module.exports = async function (
   repositoryId = String(repositoryId)
 
   const { installations, repositories } = await dbs()
+  const logs = dbs.getLogsDB()
+  const log = Log({logsDb: logs, accountId, repoSlug: repodoc.fullName, context: 'send-stale-initial-pr-reminder'})
```

Comment:
`repoDoc` needs to be fetched first

Reply:
Or rather, log needs to be initialised below `const repository = await repositories.get(repositoryId)` on line 18